### ISSUE TITLE
[REQUEST] Add a routine to automatically launch parsing and save to steam

### ISSUE THREAD

**Issue by moi952** [doZennn/steam-rom-manager#736]
- **Feature information**

I would like to not have to manually launch the parsing and adding/removing games to steam, I would like the program to detect if a game is added or removed.

Would it be possible to automate?

**Comment by DomDomonom** [comment]
Yeah, I would utterly love this too. SRM is such a powerful tool as is, and it would be wonderful if it could just do it's thing without maintenance.
